gammapy/irf/tests/test_rad_max.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import astropy.units as u
+from gammapy.maps import MapAxis
+
+
+def test_rad_max_roundtrip(tmp_path):
+    from gammapy.irf import RadMax2D
+
+    n_energy = 10
+    energy_edges = np.geomspace(100 * u.GeV, 100 * u.TeV, n_energy + 1)
+    energy_axis = MapAxis.from_edges(energy_edges, name='energy')
+
+    n_offset = 5
+    offset_edges = np.linspace(0, 2, n_offset + 1) * u.deg
+    offset_axis = MapAxis.from_edges(offset_edges, name="offset")
```

Comment:
You can use the `MapAxis.from_bounds` directly:
`offset_axis = MapAxis.from_bounds(0*u.deg, 2*u.deg, n_offset, name="offset", interp="lin")`

Reply:
done